### PR TITLE
Prepared for v0.3.5 release

### PR DESCRIPTION
**Pull request by korken89**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,11 +5,14 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [v0.3.5] - 2019-08-29
+
```
Comment (therealprof) [inline CHANGELOG.md:9]: I believe "Keep a CHANGELOG" mandates grouping the news items under a category.

**Review by therealprof — CHANGES_REQUESTED** [review]
Other than the CHANGELOG nit LGTM.

**Comment by korken89** [comment]
@therealprof Updated, please have a look

**Review by therealprof — APPROVED** [review]
LGTM, thanks!

**Pull request merged**